{"id":32,"summary":"Mockito can't create mock on public class that extends package-private class","description": "Even if it can't be implemented, I think that mockito should throw some normal exception at time of creation.\nIn my variant on first creation it returns wrong-working mock (invokes real method instead of stubbed). On second creation throws exception that doesn't really connected with problem.\nEverything works fine if you mock package-private parent."}